src/client/components/ButtonV2/ButtonV2.stories.js

```diff
@@ -27,3 +27,15 @@ ExploreBtnComponent.args = {
   onClick: action('You have clicked the button'),
   backgroundColor: 'gray',
 };
+
+// storybook-controls works with spread, but `Prop spreading is forbidden eslint(react/jsx-props-no-spreading)`
+// eslint-disable-next-line
+export const KeepShoppingBtnComponent = (args) => (<ButtonComponent2 {...args} />
+);
+KeepShoppingBtnComponent.args = {
+  title: 'KEEP SHOPPING',
+  onClick: action('You have clicked the button'),
+  backgroundColor: 'white',
+  color: '#8E0EF2',
+  border: 'solid 1px',
+};
```

Comment:
Why not just declare the args before the component declaration and then pass it as props?

Reply:
There are 3 components in this story.. so its easier to understand to user how args are in that particular component..so they can give that css and funcs..and its easy to see component in one line